nsis/gvim.nsi

```diff
@@ -1070,6 +1110,9 @@ SectionEnd
 	  RMDir ${dir}\vimfiles\keymap
 	  RMDir ${dir}\vimfiles\plugin
 	  RMDir ${dir}\vimfiles\syntax
+	  ${If} ${FileExists} $0\vimfiles\.netrwhist*
+	    Delete $0\vimfiles\.netrwhist*
```

Comment:
Is there any reason that `$0` is used instead of `${dir}`?

Reply:
You're right. This is when I moved from the "id_unsection_plugin_home" section, I did not fix it.
Thanks! I'll fix it now.